libs/animation/animation-old.ts

```diff
@@ -0,0 +1,178 @@
+/*
+    Animation library for sprites
+*/
+//% color="#03AA74" weight=78 icon="\uf021"
+namespace animation {
+    //Handles all the updates
+    let animations: OldAnimation[];
```

Comment:
IMO, this should be a new animation library (other name) which we keep the old one untouched for legacy.

Reply:
We discussed this before, and decided to keep everything in the same library. I've tested for backwards compatibility and the old blocks will still work but aren't able to be added. We also can't re-namespace it or the old blocks would be destroyed.